kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/CustomResourceDefinitionContext.java

```diff
@@ -62,6 +62,13 @@ public boolean isStatusSubresource() {
     return statusSubresource;
   }
 
+  public boolean isNamespaceScoped() {
+    if (scope != null) {
```

Comment:
Please use the constant here:
https://github.com/fabric8io/kubernetes-client/blob/a6e3cdc33faac1442aa8962a161b2dc8679d7ec9/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RawCustomResourceOperationsImpl.java#L100